Mesh_2/include/CGAL/lloyd_optimize_mesh_2.h

```diff
@@ -40,27 +41,13 @@
 #pragma GCC diagnostic ignored "-Wunused-parameter"
 #endif
 
-#ifdef BOOST_PARAMETER_MAX_ARITY
-#  if (BOOST_PARAMETER_MAX_ARITY < 8)
-#    error "BOOST_PARAMETER_MAX_ARITY must be at least 8 for CGAL::lloyd_optimize_mesh_2()"
-#  endif
-#else
-#  define  BOOST_PARAMETER_MAX_ARITY 8
-#endif
-#include <boost/parameter.hpp>
-#include <boost/parameter/name.hpp>
-
 namespace CGAL
 {
 namespace parameters
 {
   
 
 BOOST_PARAMETER_NAME( cdt )
-BOOST_PARAMETER_NAME( (max_iteration_number, tag) max_iteration_number_ )
-BOOST_PARAMETER_NAME( (convergence, tag) convergence_)
-BOOST_PARAMETER_NAME( (time_limit, tag) time_limit_ )
-BOOST_PARAMETER_NAME( (freeze_bound, tag) freeze_bound_)
 BOOST_PARAMETER_NAME( (seeds_begin, tag) seeds_begin_)
 BOOST_PARAMETER_NAME( (seeds_end, tag) seeds_end_)
 BOOST_PARAMETER_NAME( (mark, tag) mark_)
```

Comment:
All the named parameters were regrouped in /BGL even if they are only used in a single function of PMP. Maybe it ought to be the same with the parameters.

Reply:
I do not understand you choose BGL.

For PMP, there was a reason: you are using the named parameters of the BGL, but for Mesh_2 and Mesh_3 we use the named parameters from the Boost Parameters library.